package.json

```diff
@@ -33,7 +33,7 @@
     "@metamask/controller-utils": "^11.0.0",
     "@metamask/eth-json-rpc-provider": "^4.1.0",
     "@metamask/eth-query": "^4.0.0",
-    "@metamask/network-controller": "^21.1.0",
+    "@metamask/network-controller": "^22.0.0",
```

Comment:
This bump was made in this PR as well: https://github.com/MetaMask/smart-transactions-controller/pull/471